ksqldb-functional-tests/src/test/resources/historical_plans/reduce-map_-_reduce_map_with_null_initial_state/7.0.0_1614328638924/spec.json

```diff
@@ -0,0 +1,113 @@
+{
```

Comment:
can we delete the old historical plans? no need to regularly test things that never shipped (for follow up PR)

Reply:
deleted the old ones.